_posts/2019-04-07-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: news
+title: Team TLC Wins Student Robotics 2019
+---
+
+{% include figure.html
+           src="/images/content/news/sr2019/sr2019-photo.jpg"
+           caption="All the teams from SR2019" %}
+
+
+That's a wrap on Student Robotics 2019! Our teams have been working hard since November to design, build, and test their robots; finally seeing the fruits of their work on the competition weekend. It was great to see our competitors quickly adapting their robots, from the physical design to the software, to overcome unforeseen challenges; giving themselves every small advantage towards claiming victory.
+
+The Challenge: Caldera
+----------------------
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/arena.jpg"
+           caption="The arena with a raised volcano area" %}
+
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/tlc-launcher.jpg"
+           caption="Team TLC's token launcher" %}
+
+           
+{% include figure.html
+           class="right"
+           src="/images/content/news/sr2019/swi.jpg"
+           caption="Team SWI's volcano climbing robot in the Caldera" %}
+
+Our game this year, Caldera, challenged teams to control scoring zones with tokens while navigating an arena with raised platforms. Twenty-five scoring zones were arranged in a grid, varying in value from two points for each of the outermost 16 zones, a ring of eight on a raised “volcano” worth seven points apiece, to the the central “caldera” which was worth thirty points.
+
+There were a range of inventive designs this year. Many teams opted for tank tracks as a method of climbing the volcano. Instead of climbing the volcano, a few teams built catapult mechanisms with varying degrees of success. Others scooped up tokens and deposited them on the low scoring areas and/or the edge of the volcano. Some teams lifted tokens onto top-mounted conveyor belts which dropped them onto the volcano's surface.
+
+Some of the more notable designs were:
+
+- Team TLC with their consistent, successful catapult.
+- Team SWI with their simple and effective grab-and-go robot which climbed the volcano with a single token.
+- Team CRB with their suction cup and catapult combination which unfortunately had technical issues for much of the competition.
+- Team HAB with their Hawaiian themed robot, topped with a 3D paper pineapple.
+
+The Final
+---------
+
+Student Robotics 2019 concluded with an intense match between PSC, HAM, TLC, and HRS. Team HAM were quick off the mark pushing their tokens into the scoring zones. Team HRS soon followed, violently shoving their tokens across the arena. Team PSC's robot slowly moved 4 of their tokens into a single scoring zone before unfortunately flipping onto its back. Meanwhile, Team TLC quietly moved towards the edge of the volcano and launched a token directly into the Caldera.
```

Comment:
Since we're a non-violent competition (and repeatedly need to encourage people not to compare us to robot wards), perhaps use a word other than "violently" here?
I didn't actually see the match, so I don't know how appropriate the description is, however if it was truly violent then I'd question why they weren't disqualified.